index.html

```diff
@@ -266,19 +266,21 @@ <h3 class="intro">
             </h3>
             </div>
 
-            <form id="contact-form" action="https://formspree.io/f/mrgjoqgg" method="post">
-                <ul>
-                    <li>
+            <form id="contact-form" action="https://formspree.io/f/xbjwqkbk" method="post">
+                <ul class="input-section">
+                    <li class="input-control form-info">
+                        <label for="text"></label>
                         
-                        <input type="text" id="text" placeholder="Full Name" name="User_name" maxlength="30" required>
+                        <input type="text" id="text" placeholder="Full Name" name="User_name" maxlength="30">
                     </li>
 
-                    <li>
-                       <input type="email" id="email" placeholder="E-mail address" name="e-mail" required>
+                    <li class="input-control form-info">
+                        <label for="email"></label>
+                       <input type="email" id="email" placeholder="E-mail address" name="e-mail">
                     </li>
```

Comment:
- [ ] Kindly make sure the error message is displayed near the submit button(below or above it) currently its below the email field.
<img width="560" alt="image" src="https://user-images.githubusercontent.com/61408860/168308893-0fb83467-6ec8-4519-8645-b0681ad8bb04.png">